pyodide-build/pyodide_build/buildpkg.py

```diff
@@ -473,6 +475,7 @@ def package_wheel(
     build_metadata: _BuildSpec,
     bash_runner: BashRunnerWithSharedEnvironment,
     host_install_dir: str,
+    extract_metadata_file: bool,
```

Comment:
Can probably give it a default value,
```suggestion
    extract_metadata_file: bool = False
```